src/i18n/en-US/articles/governanceReviewTeam.ts

```diff
@@ -1,3 +1,41 @@
+import { SystemIntakeActionType } from 'types/graphql-global-types';
+
+/** System intake action type translations */
+const actionNameTranslations: Record<SystemIntakeActionType, string> = {
```

Comment:
Love the use of `Record` here -- helps ensure we don't miss any translations going forward!